test/run

```diff
@@ -25,18 +25,28 @@ test_nodemon_removed
 test_npm_cache_cleared
 test_npm_tmp_cleared
 kill_test_application
-test_dev_mode_app_true_development
-test_dev_mode_app_false_production
+test_dev_mode_true_development
+test_dev_mode_false_production
 "
 
-TEST_LIST_DEV="\
+TEST_LIST_NODE_ENV="\
 test_run_app_application
 test_connection
 test_nodemon_present
 test_npm_cache_exists
 kill_test_application
-test_dev_mode_app_true_development
-test_dev_mode_app_false_development
+test_dev_mode_true_development
+test_dev_mode_false_development
+"
+
+TEST_LIST_DEV_MODE="\
```

Comment:
TEST_LIST_DEV_MODE and TEST_LIST_DEV_MODE are same is it right? Otherwise LGTM

Reply:
Not really. `TEST_LIST_NODE_ENV` contains a test `test_dev_mode_false_development` at the end but `TEST_LIST_DEV_MODE` contains a test `test_dev_mode_false_production` at the end.

Reply:
ok thanks so we can merge it